src/psyclone/dynamo0p3_builtins.py

```diff
@@ -658,16 +686,16 @@ def gen_code(self, parent):
 
 
 # ------------------------------------------------------------------- #
-# ============== Multiplying (scaled) fields ======================== #
+# ============== Multiplying (scaled) real fields =================== #
 # ------------------------------------------------------------------- #
 
 
 class DynXTimesYKern(DynBuiltIn):
-    ''' DoF-wise product of one field with another with the result
-    returned as a third field '''
+    ''' DoF-wise product of one real-valued field with another with
+    the result returned as a third real-valued field '''
```

Comment:
Full stop and move the blank line please.

Reply:
Yep, done for all such docstrings.